pkg/domainutil/domain.go

```diff
@@ -0,0 +1,59 @@
+package domainutil
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+const (
+	DefaultDomain = "astronomer.io"
+	LocalDomain   = "localhost"
+)
+
+var (
+	// CloudDomainRegex is used to differentiate cloud domain from software domain
+	// See https://github.com/astronomer/astrohub-cli/issues/7 for regexp rationale
+	// This will need to be handled as part of the permanent solution to issue #432
+	CloudDomainRegex     = regexp.MustCompile(`(cloud\.|^)astronomer(?:(-dev|-stage|-perf))?\.io`)
+	PRPreviewDomainRegex = regexp.MustCompile(`(pr\d{4}\.)astronomer(-dev)\.io$`)
```

Comment:
4+

Reply:
`^(pr\d{4+}\).astronomer-dev\.io$`
`matches := re.FindStringSubmatch` would get you one result here

Reply:
I changed it to `^(pr\d{4,6}).astronomer-dev\.io$` so that it still has an upper limit on the numbers.